react/src/util/api-client.ts

```diff
@@ -1,14 +1,37 @@
 import axios from 'axios';
 
+// export const getAddressDetails = async (
+//   address: string,
+// ): Promise<AddressDetails> => {
+//   const res = await fetch(
+//     `https://rest.bitcoin.com/v2/address/details/${address}`,
+//   );
+//   return await res.json();
+// };
+
 export const getAddressDetails = async (
   address: string,
 ): Promise<AddressDetails> => {
   const res = await fetch(
-    `https://rest.bitcoin.com/v2/address/details/${address}`,
+    `https://api.paybutton.org/address/transactions/${address}`,
   );
   return await res.json();
 };
 
+export const getSatoshiBalance = async (
+  address: string,
+): Promise<{ satoshis: number }> => {
+  const res = await fetch(
+    `https://api.paybutton.org/address/balance/${address}`,
```

Comment:
Can we move `https://api.paybutton.org/` to a parameter where the default value is `https://api.paybutton.org/` ?

That way it's all in 1 place and can be changed easily by people who decide to run their own server.